test/test_runner.sh

```diff
@@ -33,4 +33,4 @@ TESTS=("${@:-.}")
 export JOBS=${JOBS:-$(($(nproc --all) * 4))}
 
 # Run the tests.
-execute bats --jobs "$JOBS" --tap "${TESTS[@]}"
+execute bats --no-parallelize-within-files --jobs "$JOBS" --tap "${TESTS[@]}"
```

Comment:
We need serial execution because tests need to deal with files in `/etc`, and this is global state. Fine.
I think we should have a separate serial runner though, or some option to NOT slow down every other test, which can and should execute in parallel.

Reply:
I have changed the configuration so only the irqbalance tests are serial executed.